src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -353,12 +354,13 @@ void DlgPrefLibrary::slotSelectFont() {
 }
 
 void DlgPrefLibrary::slotSyncTrackMetadataExportToggled() {
-    if (isVisible() && checkBox_SyncTrackMetadataExport->isChecked()) {
+    if (isVisible() && checkBox_SyncTrackMetadataExport->isChecked() && m_bShowTrackMetadataExportInfo) {
         QMessageBox::information(
                 nullptr,
                 tr("Export Modified Track Metadata"),
                 tr("File modifications are deferred and may not appear at once! "
```

Comment:
How about: "Mixxx may wait to modify files until it is certain that changing the files will not corrupt them. If you do not see changed metadata in other programs immediately, close Mixxx to have your changes saved to the files."